requirements.txt

```diff
@@ -0,0 +1,29 @@
+-i https://pypi.org/simple
```

Comment:
@fridex @goern I think we need regenerate this file, right? and get rid of this line?

Reply:
Why do you want to remove this line?

Reply:
Oh, I misunderstood the previous PR of yours. Let's merge this to address #257.